MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -689,189 +628,300 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, VideoOptions options)
-                                break;
-                            }
-                        }
-
-                        if (applyConditions)
-                        {
-                            var transcodingAudioCodecs = ContainerProfile.SplitValue(transcodingProfile.AudioCodec);
-                            foreach (var transcodingAudioCodec in transcodingAudioCodecs)
-                            {
-                                if (i.ContainsAnyCodec(transcodingAudioCodec, transcodingProfile.Container))
-                                {
-                                    ApplyTranscodingConditions(playlistItem, i.Conditions, transcodingAudioCodec, true, isFirstAppliedCodecProfile);
-                                    isFirstAppliedCodecProfile = false;
-                                }
-                            }
-                        }
+                        ApplyTranscodingConditions(playlistItem, i.Conditions, transcodingAudioCodec, true, isFirstAppliedCodecProfile);
+                        isFirstAppliedCodecProfile = false;
+                        break;
                     }
                 }
+            }
 
-                var maxBitrateSetting = options.GetMaxBitrate(false);
-                // Honor max rate
-                if (maxBitrateSetting.HasValue)
-                {
-                    var availableBitrateForVideo = maxBitrateSetting.Value;
-
-                    if (playlistItem.AudioBitrate.HasValue)
-                    {
-                        availableBitrateForVideo -= playlistItem.AudioBitrate.Value;
-                    }
+            var maxBitrateSetting = options.GetMaxBitrate(false);
+            // Honor max rate
+            if (maxBitrateSetting.HasValue)
+            {
+                var availableBitrateForVideo = maxBitrateSetting.Value;
 
-                    // Make sure the video bitrate is lower than bitrate settings but at least 64k
-                    long currentValue = playlistItem.VideoBitrate ?? availableBitrateForVideo;
-                    var longBitrate = Math.Max(Math.Min(availableBitrateForVideo, currentValue), 64000);
-                    playlistItem.VideoBitrate = longBitrate >= int.MaxValue ? int.MaxValue : Convert.ToInt32(longBitrate);
+                if (playlistItem.AudioBitrate.HasValue)
+                {
+                    availableBitrateForVideo -= playlistItem.AudioBitrate.Value;
                 }
-            }
 
-            playlistItem.TranscodeReasons = transcodeReasons.ToArray();
+                // Make sure the video bitrate is lower than bitrate settings but at least 64k
+                long currentValue = playlistItem.VideoBitrate ?? availableBitrateForVideo;
+                var longBitrate = Math.Max(Math.Min(availableBitrateForVideo, currentValue), 64000);
+                playlistItem.VideoBitrate = longBitrate >= int.MaxValue ? int.MaxValue : Convert.ToInt32(longBitrate);
```

Comment:
Not sure about the `Convert.ToInt32`, only thing it does it trow if `longBitrate < int.MinValue`